sources/packages/backend/libs/services/src/student-loan-balance/student-loan-balance-shared.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { StudentLoanBalance } from "@sims/sims-db";
+import { Repository } from "typeorm";
+@Injectable()
```

Comment:
Please add a blank line above the `@Injectable()` following what is present for most of the files.